plugins/kmeans.cc

```diff
@@ -197,7 +198,15 @@ run(const ProcedureRunConfig & run,
     if (!runProcConf.modelFileUrl.empty()) {
         try {
             Datacratic::makeUriDirectory(runProcConf.modelFileUrl.toString());
-            kmeans.save(runProcConf.modelFileUrl.toString());
+            Json::Value md;
```

Comment:
I see.  I suggest we delete the old serialization code then to avoid further problem.